### PR TITLE
ECIP 1103: Magneto EVM and Protocol Upgrades

### PR DESCRIPTION
**Pull request by q9f**

### CODE REVIEW

**Review by TokenHash — APPROVED** [review]

**Comment by TokenHash** [comment]
lgtm

**Review by gitr0n1n — APPROVED** [review]
LGTM.

**Comment by gitr0n1n** [comment]
@q9f Does Mantis support Berlin upgrades as well? Not sure if they've been pinged. @bpmckenna

**Comment by bobsummerwill** [comment]
@gitr0n1n There is not Berlin support in Mantis, no.   Unlike the other clients, Mantis is ETC specific, so there is no upstream to pull from.

Also worth knowing is that Berlin is not even activated on ETH Mainnet yet.

**Comment by gitr0n1n** [comment]
> @gitr0n1n There is not Berlin support in Mantis, no. Unlike the other clients, Mantis is ETC specific, so there is no upstream to pull from.
> 
> Also worth knowing is that Berlin is not even activated on ETH Mainnet yet.

Let me rephrase. This PR state Besu and Core-Geth intend to support Berlin upgrades. Does Mantis also intend to support Berlin EIPs?

**Review by developerkevin — APPROVED** [review]
LGTM.
